File: app/src/main/java/com/anahorn/fukusizer/IconHelpers.java
package com.anahorn.fukusizer;

import java.util.HashMap;
import java.util.Map;

public class IconHelpers {
    protected static Map<String, Integer> getMapOfFlagIcons() {
        Map<String, Integer> drawableMap = new HashMap<>();
        drawableMap.put("Australia", R.drawable.australia);
        drawableMap.put("Belgium", R.drawable.belgium);
        drawableMap.put("China", R.drawable.china);
        drawableMap.put("Europe", R.drawable.europe);
        drawableMap.put("France", R.drawable.france);
        drawableMap.put("Germany", R.drawable.germany);
        drawableMap.put("India", R.drawable.india);
        drawableMap.put("Italy", R.drawable.italy);
        drawableMap.put("Japan", R.drawable.japan);
        drawableMap.put("Korea", R.drawable.korea);
        drawableMap.put("New Zealand", R.drawable.newzealand);
        drawableMap.put("Spain", R.drawable.spain);
        drawableMap.put("UK", R.drawable.uk);
        drawableMap.put("USA", R.drawable.usa);
        return drawableMap;
    }

    protected static Map<String, Integer> getMapOfWomenClothingIcons() {
        Map<String, Integer> drawableMap = new HashMap<>();
        drawableMap.put("Tops", R.drawable.tops_women);
        drawableMap.put("Bottoms", R.drawable.bottoms_women);
        drawableMap.put("Dresses", R.drawable.dresses);
        drawableMap.put("Outerwear", R.drawable.outerwear);
        drawableMap.put("Shoes", R.drawable.shoes);
        drawableMap.put("Bra", R.drawable.bra);
        drawableMap.put("Panties", R.drawable.panties_women);
        drawableMap.put("Pantyhose", R.drawable.pantyhose);
        drawableMap.put("Swimwear", R.drawable.swimwear_women);
        drawableMap.put("Hats", R.drawable.hats_women);
        return drawableMap;
    }

    protected static Map<String, Integer> getMapOfMenClothingIcons() {
        Map<String, Integer> drawableMap = new HashMap<>();
        drawableMap.put("Tops", R.drawable.tops_men);
        drawableMap.put("Bottoms", R.drawable.bottoms_men);
        drawableMap.put("Suiting/Jackets", R.drawable.suiting_jackets_men);
        drawableMap.put("Suiting/Pants", R.drawable.bottoms_men);
        drawableMap.put("Outerwear", R.drawable.outerwear);
        drawableMap.put("Shoes", R.drawable.shoes);
        drawableMap.put("Hats", R.drawable.hats_men);
        return drawableMap;
    }
}

File: app/src/main/java/com/anahorn/fukusizer/IconHelpers.kt
package com.anahorn.fukusizer

object IconHelpers {
    fun getMapOfFlagIcons(): Map<String, Int> {
        return mapOf(
            "Australia" to R.drawable.australia,
            "Belgium" to R.drawable.belgium,
            "China" to R.drawable.china,
            "Europe" to R.drawable.europe,
            "France" to R.drawable.france,
            "Germany" to R.drawable.germany,
            "India" to R.drawable.india,
            "Italy" to R.drawable.italy,
            "Japan" to R.drawable.japan,
            "Korea" to R.drawable.korea,
            "New Zealand" to R.drawable.newzealand,
            "Spain" to R.drawable.spain,
            "UK" to R.drawable.uk,
            "USA" to R.drawable.usa
        )
    }

    fun getMapOfWomenClothingIcons(): Map<String, Int> {
        return mapOf(
            "Tops" to R.drawable.tops_women,
            "Bottoms" to R.drawable.bottoms_women,
            "Dresses" to R.drawable.dresses,
            "Outerwear" to R.drawable.outerwear,
            "Shoes" to R.drawable.shoes,
            "Bra" to R.drawable.bra,
            "Panties" to R.drawable.panties_women,
            "Pantyhose" to R.drawable.pantyhose,
            "Swimwear" to R.drawable.swimwear_women,
            "Hats" to R.drawable.hats_women
        )
    }

    fun getMapOfMenClothingIcons(): Map<String, Int> {
        return mapOf(
            "Tops" to R.drawable.tops_men,
            "Bottoms" to R.drawable.bottoms_men,
            "Suiting/Jackets" to R.drawable.suiting_jackets_men,
            "Suiting/Pants" to R.drawable.bottoms_men,
            "Outerwear" to R.drawable.outerwear,
            "Shoes" to R.drawable.shoes,
            "Hats" to R.drawable.hats_men
        )
    }
}

